### PR TITLE
Auto Scroll to Planner Helper

### PR DESCRIPTION
**Pull request by krau612**
Added two lines of code to the on click function for the "View Data" button so the page auto scrolls to the Planner Helper data.

### CODE REVIEW

File: src/inject/planner_helper.js [hunk 1]
```diff
@@ -86,6 +86,8 @@ PlannerHelper.prototype.makeViewDataButton = function (teacher, course) {
     button.data('teacher', teacher);
     button.on('click', function () {
         plannerHelper.reloadData($(this).data('teacher'), $(this).data('course'));
+        var elem = document.getElementById('planner-helper');
```
Comment (j-delaney) [inline src/inject/planner_helper.js]: For consistency with the rest of the code could you use a jQuery selector instead?

```
var elem = $('#planner-helper')
```

Comment (j-delaney) [inline src/inject/planner_helper.js]: Actually, I think this should be making use of instance variables. If you look at [the code](https://github.com/krau612/planner-helper/blob/216e1a099005f9bbcd0e6be422f0374bea887ba7/src/inject/planner_helper.js#L11) you'll notice that there is a variable `this.element`

**Comment by krau612** [comment]
I updated it to use JQuery. I couldn't get it to work using "this.element" though.

**Comment by j-delaney** [comment]
Thanks for contributing!

**Pull request merged**